.github/workflows/boulder-ci.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         # Add additional docker image tags here and all tests will be run with the additional image.
         BOULDER_TOOLS_TAG:
           - go1.17.7_2022-03-08
-          - go1.18beta2_2022-03-08
+          - go1.18_2022-03-08
```

Comment:
I think you want to change both of these to `go1.18_2022-03-15`. I don't believe there was an arm64 build pushed on 2022-03-08.

Reply:
Ah, the tag for the 2022-03-08 was `go1.18beta2_2022-03-08`. So yeah, you need to bump this to the tag I suggested above.